### ISSUE TITLE
SK dll causes Dragons Dogma 2 to crash

### ISSUE THREAD

**Issue by R1fast** [SpecialKO/SpecialK#155]
Guessing this is due to Crapcoms Denuvo implementation, and not sure a fix is even possible, but figured I would add it for tracking as I didn't see it in the Issues list.

**Comment by marat569** [comment]
Install Reframework ( https://www.nexusmods.com/dragonsdogma2/mods/8 ) and SK will work perfectly fine!

**Comment by Kaldaien** [comment]
Nah, nothing to do with Denuvo. It's CAPCOM's usual anti-tamper. Updated version of REFramework removes just enough of the anti-tamper to allow mods like SK to work, while keeping enough of it there to (hopefully) not anger CAPCOM into adding even more obnoxious anti-tamper.

Shame it couldn't -all- be removed, because if it were, performance would be much better.
